gulpfile.js/index.js

```diff
@@ -36,7 +38,7 @@ function collectRevisionsIntoManifest(cb) {
 // rewrite source HTML files with build versioned files and assets
 // usually run as the final step linking the build together
 function templateHTMLForProduction() {
-  const manifest = src(`${CLIENT_FOLDER}/rev-manifest.json`);
+  const manifest = readFileSync(`${CLIENT_FOLDER}/rev-manifest.json`);
   return src('client/src/index.html')
```

Comment:
I don't understand why the "src" of rev-manifest.json did not work, but the "src" of client/src/index.html did.

Reply:
The technical explanation is that `src()` creates a NodeJS stream, whereas `readFileSync()` creates a Buffer.  All of gulp works by piping streams from a `src()` to a `dest()`.  However, their documentation recommends against plugins taking in a `src()` - all plugins should use JSON/Buffer/Strings/Numbers/etc as parameters.  `rev-revision` recently updated to be compliant and it breaks this.

Why?  I have no idea.  I assume there is a good explanation for it, but I haven't bothered to check.